deposit/src/main/resources/service-context.xml

```diff
@@ -32,6 +33,15 @@
 		</property>
 		<property name="ignoreResourceNotFound" value="false" />
 	</bean>
+	
+	<bean id="cdrVersion" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
```

Comment:
change the id to indicate that this is not actually the cdrVersion, but is something that injects the cdrVersion into the software agents class